_posts/en/2019-12-20-why-we-are-promoting-risks.md

```diff
@@ -0,0 +1,131 @@
+Getting information on the scope of change *is* also work.
+It requires re-engaging the different IT teams, stopping them from doing what they are currently doing, and re-evaluating the same type of information as the original project estimates (scope, costs, and schedule).
+
+As per above, the effort of going through the change control process must be proportional to the actual change effort, otherwise it gets piled up.
+This effort also includes going through governance bodies and having to explain why we could not have predicted such a change in the first place; why our original estimates were not accurate.
+Explaining why we were wrong is not fun...
+
+In a sense, the current governance does not welcome change, which is a direct contradict to the [Manifesto for Agile Software Development](https://agilemanifesto.org/) (4th value in the list).
+
+Finally, because the **Project Sponsor** already committed to a fixed scope, costs, and schedule at gate 3, and because change is actually difficult to implement, we end up sticking to the plan, especially scope and schedule.
+This results in IT having to cut essential work that is not visible to the sponsor such as testing, quality assurance, performance, security, documentation, and most especially: technical debt remediation.
+The project may be delivered on time and on budget but so much cutting has been made that its quality suffers and the IT team ends up having to absorb a large amount of debt.
+Some of this debt will be articulated in a residual risk plan that security will use to issue its interim Authority To Operate with a promise that the plan will be executed in the future.
+Which, let's be honest, ends up unlikely because this unattractive, non-functional requirements, plan will be competing with attractive new-feature-rich plans that have been waiting for their turn.
+
+### Incorrect investment decisions
+
+In the PMLC mentioned above, **Develop and test solution** is only one project activity from the list of 48 to 60 but is the most intensive one, the reason this whole project was funded, where we will spend most of our time, and will involve at least 12 more stakeholders just within the IT Branch (as per [this process map](onenote:http://dialogue/grp/PR6893344/OneNote/AppDevSA/02-Development%20(In%20Progress)/Outcomes%20Map.one#Feature%20to%20Release%20BPM&section-id={5E16E60C-310B-49EF-8451-88E0CE4DA968}&page-id={8208D63D-CE7D-49EB-BF48-033A38FA5825}&end) created by our Senior Advisors).
+
+It is in this **develop and test solution** activity that we end up uncovering many unknowns (internal skill sets limitations, systems integration limitations, technical debt, new technology stack capabilities that could solve problems but are not part of our current standards, security vulnerabilities, etc.).
```

Comment:
Will add about uncovering proprietary software limitations and hitting the procurement wall should we need to engage the vendor.

If we planned ahead of time to have a vendor "sell us" a solution it would be in the plannig phase. But great point on not being able to course correct easily if we chose proprietary software compared to open source ones.